lib/hackerone/client.rb

```diff
@@ -66,17 +77,20 @@ def reporters
         end
       end
 
-      ## Returns all open reports, optionally with a time bound
+      ## Returns all reports in a given state, optionally with a time bound
       #
       # program: the HackerOne program to search on (configure globally with Hackerone::Client.program=)
       # since (optional): a time bound, don't include reports earlier than +since+. Must be a DateTime object.
+      # state (optional): state that a report is in, by default new
       #
       # returns all open reports or an empty array
-      def reports(since: 3.days.ago)
+      def reports(since: 3.days.ago, state: "new")
         raise ArgumentError, "Program cannot be nil" unless program
+        raise ArgumentError, "State is invalid" unless REPORT_STATES.include? state
```

Comment:
We should probably accept symbols (and should have on the previous PR too)

```suggestion
        raise ArgumentError, "State is invalid" unless REPORT_STATES.include?(state.to_s)
```